### PR TITLE
fix: global config in ticket header

### PR DESCRIPTION
**Pull request by Rom1-B**
### Changes description

The global configuration appeared in the ticket header. This was not consistent.

Before:
image

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### References

Close: #N/A

### CODE REVIEW

**Comment by cedric-anne** [comment]
This is not the global configuration, but the default configuration for current ticket.

It has been done intentionnaly in #60. A solution could be to introduce a dedicated right for this, so people may disable this right to not see these fields.

**Review by cedric-anne — CHANGES_REQUESTED** [review]
See my previous comment.

File: inc/ticketconfig.class.php [hunk 1]
```diff
@@ -83,6 +86,12 @@ static function showForTicket(Ticket $ticket, bool $embed_in_ticket_form = false
       if (!Session::haveRight("entity", UPDATE)) {
          return;
       }
```
Comment (cedric-anne) [inline inc/ticketconfig.class.php:88]: We should remove this.
```suggestion
```

File: inc/ticketconfig.class.php [hunk 2]
```diff
@@ -35,7 +35,10 @@
 
 class PluginCreditTicketConfig extends CommonDBTM {
 
-   public static $rightname = 'entity';
+   public static $rightname = 'plugin_creditticketconfig';
+
+   const TICKET_TAB  = 1024;
+   const TICKET_FORM = 2048;
```
Comment (cedric-anne) [inline inc/ticketconfig.class.php:41]: A migration should be added to add `plugin_creditticketconfig:TICKET_TAB&TICKET_FORM` right when `plugin_creditticketconfig` does not exist yet and user has the `entity:UPDATE` right.

The `Migration::addRight()` method could probably be used for this.

**Review by cedric-anne — CHANGES_REQUESTED** [review]

**Comment by Rom1-B** [comment]
Replaced by https://github.com/pluginsGLPI/credit/pull/96

**Pull request closed without merging**